### PR TITLE
[modified] web models structure for page object tests implementation

### PR DESCRIPTION
**Pull request by dimencionrider**

### CODE REVIEW

File: src/main/java/core/modules/web/models/Web.java [hunk 1]
```diff
@@ -38,76 +38,161 @@ public Web openAnyUrl(String url) {
...
+        return this;
+    }
+
     public Web useXpathElementForAssertionAnd(String xpath) {
         WebElement element = findElementByXpath(xpath);
         return this;
     }
 
+    public <T extends Enum> WebAssertion useXpathElementsForAssertion(T xpath) {
+        List<WebElement> elements = driver.findElements(By.xpath(String.valueOf(xpath)));
```
Comment (Antonyan) [inline src/main/java/core/modules/web/models/Web.java:130]: Is this case that you were telling about? when we have a list of elements and want to use one of them

File: src/main/java/core/modules/web/models/WebAssertion.java [hunk 2]
```diff
@@ -58,6 +54,31 @@ public void shouldBeDisplayed(boolean condition) {
         Assert.assertFalse(elementForAssert.isDisplayed(), "Element " + elementForAssert.toString() + " is displayed");
     }
 
+    public void shouldBeVisibleByCss(boolean condition, String cssSelector) {
```
Comment (Antonyan) [inline src/main/java/core/modules/web/models/WebAssertion.java:51]: Better to use obvious names: shouldBeVisibleByCss and shouldNotBeVisibleByCss

File: src/main/java/core/modules/web/models/Web.java (hunk 1 above)
Comment (dimencionrider) [reply src/main/java/core/modules/web/models/Web.java:130]: yes, that is correct. Sometimes we will need the collection of web elements to interact with.

Comment (dimencionrider) [reply src/main/java/core/modules/web/models/Web.java:130]: So I created these methods for almost all existing cases.

File: src/main/java/core/modules/web/models/WebAssertion.java (hunk 2 above)
Comment (dimencionrider) [reply src/main/java/core/modules/web/models/WebAssertion.java:51]: I agree. I'll delete this method for now.

**Comment by dimencionrider** [comment]
please review

**Pull request merged**